cron/update_geolite_db.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Mozilla Corporation
+#
+# Contributors:
+# Brandon Myers bmyers@mozilla.com
+
+import sys
+import os
+from configlib import getConfig, OptionParser
+
+import urllib2
+import tempfile
+import tarfile
+
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../lib'))
+from utilities.logger import logger, initLogger
+
+
+def fetch_db_data(db_download_location):
+    logger.debug('Fetching db data from ' + db_download_location)
+    response = urllib2.urlopen(db_download_location)
+    db_raw_data = response.read()
+    with tempfile.NamedTemporaryFile(mode='wb') as temp:
+        logger.debug('Writing compressed gzip to temp file: ' + temp.name)
+        temp.write(db_raw_data)
+        temp.flush()
+        logger.debug('Extracting gzip data from ' + temp.name)
+        tar = tarfile.open(temp.name)
+        for tarinfo in tar:
+            if tarinfo.name.endswith('GeoLite2-City.mmdb'):
+                extracted_file = tar.extractfile(tarinfo.name)
+                return extracted_file.read()
+
+
+def save_db_data(save_path, db_data):
+    logger.debug("Saving db data to " + save_path)
+    if os.path.isfile(save_path):
+        logger.debug("Overwriting db data in " + str(save_path))
+    else:
+        logger.debug("Creating new db data file at " + str(save_path))
+    with open(save_path, "wb+") as text_file:
+        text_file.write(db_data)
```

Comment:
Added in https://github.com/mozilla/MozDef/pull/433/commits/53828fafd8b1a338eefb1a193a204b246af1eefe